main.py

```diff
@@ -1,70 +1,93 @@
-## Mad libs generator
+#!/usr/bin/env python3
```

Comment:
This is called a shebang. It means if I run your code via `./main.py` instead of `python3 main.py` it will run via python3 and not something else.

Reply:
how does it know its 3.9?

Reply:
It doesn't. It just knows it's python 3. The specific version is up to whatever you've installed. I think the type hints below are for all versions of python3